pym/bob/cmds/build/builder.py

```diff
@@ -104,6 +104,79 @@ class RestartBuildException(Exception):
 class CancelBuildException(Exception):
     pass
 
+class ExternalJobServer:
+    def __init__(self, makeFds):
+        self.__makeFds = makeFds
+
+    def getMakeFd(self):
+        return self.__makeFds
+
+class JobServer:
```

Comment:
Maybe name it `InternalJobServer` for consistency?

```suggestion
class InternalJobServer:
```